robot-core/src/main/java/org/obolibrary/robot/OntologyHelper.java

```diff
@@ -1660,11 +1662,15 @@ public static void trimOntology(OWLOntology ontology) {
    * @param ontology OWLOntology to remove axiom annotations
    * @param properties Annotation property IRIs to remove related axiom annotations.
    */
-  public static void removeAxiomAnnotations(OWLOntology ontology, List<IRI> properties) {
```

Comment:
This breaks a public interface by removing the method with this signature. We try hard to avoid that. Can the old method/signature (with `List<IRI>`) be maintained (maybe as a thin wrapper) while the new method (with `Map<IRI, String>`) is added?

Reply:
Done! Thanks you!